typescript2/pkg-editor/src/views-workbench.css

```
@@ -15,14 +15,189 @@ div[is="app"] {
+  width: 100%;
+  overflow: hidden;
+  font-family: var(--vscode-editor-font-family, monospace);
+}
+
+.baml-editor-loading-status {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+  white-space: nowrap;
+  clip: rect(0 0 0 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)